### PR TITLE
Add Cambridge Audio integration

### PR DESCRIPTION
**Pull request by noahhusby**
## Proposed change

This includes the base documentation for a new Cambridge Audio integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125642
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5845
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for integrating Cambridge Audio receivers into Home Assistant.
	- Enhanced user interaction with automatic discovery of zones and sources, represented as media player devices.
	- Support for media information display and transport controls based on source capabilities.
	- Listed compatibility with various Cambridge Audio models, including Evo 75, Evo 150, and CXN.
	- Added structured integration details such as category, release version, and configuration flow.

### CODE REVIEW

File: source/_integrations/cambridge_audio.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ha_domain: cambridge_audio
+ha_platforms:
+  - media_player
+ha_codeowners:
+  - '@noahhusby'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The `cambridge_audio` {% term integration %} allows you to control all receivers and streamers that support the StreamMagic app.
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
The **Cambridge Audio** {% term integration %} allows you to control all receivers and streamers that support the StreamMagic app.
```

File: source/_integrations/cambridge_audio.markdown [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  - media_player
+ha_codeowners:
+  - '@noahhusby'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The **Cambridge Audio** {% term integration %} allows you to control all receivers and streamers that support the StreamMagic app.
+
+The platform automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information and transport controls are supported if the selected source reports it.
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
The integration automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information and transport controls are supported if the selected source reports it.
```

Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]: what are transport controls?

File: source/_integrations/cambridge_audio.markdown [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Transport control functionality depends on the source / service that is currently selected.
+The interface automatically sets which controls are available depending on which source is selected.
+
+### The ability to change volume is missing
+
+Volume control is only supported on all-in-one amps, or streamers with pre-amp mode.
+Likely, the device is not configured to be in pre-amp mode.
+This can be changed by navigating to the IP address of the device in a web browser,
+or selecting settings in the StreamMagic app and setting `Pre-Amp` to `On`.
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
or selecting settings in the StreamMagic app and setting **Pre-Amp** to **On**.
```
We use bold to markup UI strings.

File: source/_integrations/cambridge_audio.markdown [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### Turning on the device doesn't work from Home Assistant
+
+Cambridge Audio devices come with ECO mode enabled by default, which disables the network interface when
+the device is powered down. This can be changed by navigating to the IP address of the device in a web browser,
+or selecting settings in the StreamMagic app and setting `Standby Mode` to `Network standby`.
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
or selecting settings in the StreamMagic app and setting **Standby Mode** to **Network standby**.
```

File: source/_integrations/cambridge_audio.markdown [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  - media_player
+ha_codeowners:
+  - '@noahhusby'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The **Cambridge Audio** {% term integration %} allows you to control all receivers and streamers that support the StreamMagic app.
+
+The integration automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information and transport controls are supported if the selected source reports it.
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]: What are transport controls?

File: source/_integrations/cambridge_audio.markdown [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+    description: The IP address of your device can be found by navigating to the device on the [StreamMagic app](https://www.cambridgeaudio.com/usa/en/products/streammagic) and selecting `Settings` → `IP address`.
+    required: true
+    type: string
+{% endconfiguration_basic %}
+
+## FAQ
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
## Troubleshooting
```
The topics listed below sound like troubleshooting information.

Comment (noahhusby) [reply source/_integrations/cambridge_audio.markdown]: Yep, they are. LGTM

File: source/_integrations/cambridge_audio.markdown (hunk 5 above)
Comment (noahhusby) [reply source/_integrations/cambridge_audio.markdown]: Controls like play, pause, skip, previous, and seek

Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
The integration automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information and controls (such as play, pause, skip) are supported if the selected source reports it.
```
Ah, ok. Could "transport" be omitted?

File: source/_integrations/cambridge_audio.markdown [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+### The buttons to skip, shuffle, and repeat the track are missing
+
+Transport control functionality depends on the source / service that is currently selected.
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
Control functionality depends on the source / service that is currently selected.
```

File: source/_integrations/cambridge_audio.markdown (hunk 5 above)
Comment (noahhusby) [reply source/_integrations/cambridge_audio.markdown]: Yes, that's fine.

File: source/_integrations/cambridge_audio.markdown [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+    description: The IP address of your device can be found by navigating to the device on the [StreamMagic app](https://www.cambridgeaudio.com/usa/en/products/streammagic) and selecting `Settings` → `IP address`.
+    required: true
+    type: string
+{% endconfiguration_basic %}
+
+## FAQ
```
Comment (c0ffeeca7) [inline source/_integrations/cambridge_audio.markdown]:
```suggestion
## Troubleshooting
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @noahhusby :+1:

**Pull request merged**